LiveSplit.AutoSplitters.xml

```diff
@@ -16676,5 +16676,15 @@
         </URLs>
         <Type>Script</Type>
         <Description>ASL script for the 1989 C64 game 'Rick Dangerous'. Meant for use with Vice 3.6.1</Description>
-    </AutoSplitter>    
+    </AutoSplitter>
+    <AutoSplitter>
+        <Games>
+            <Game>Jolly 3: Chapter 1</Game>
+        </Games>
+        <URLs>
+            <URL>https://raw.githubusercontent.com/derric-young/Jolly3SpeedrunStuff/main/Jolly%203%20Chapter%201%20Auto%20Splitter.asl</URL>
+        </URLs>
+        <Type>Script</Type>
+        <Description>Start and Split by throwaway acount. Reset by derric. on night 1. dont press x and have the timer set to -28.59</Description>
```

Comment:
What's going on with this description?

Reply:
im going to go with 1. the autosplitter desciption. explains that the autosplitter was initally created by AITYunivers but there were a couple of problems. 1. it asked for the exe name jolly3 instead of jolly3-v1.0.3 (the current version you can get. if i find a older version. ill simply update it with older version exe file names) and 2. never decided to add the autosplitter to this database assuming the file ACUALLY GETS UPLOADED HERE instead of it being linked. and 3. no reset option. so i forked it and added and changed what was needed and this is of that. i made the reset funtion by looking for if frame 28 (the frame that just plays a video file) is loaded and resets the timer. also the description mentions the timer MUST BE -28.59 as timing starts upon turning 12 AM in night 1. yes. its a fnaf fan game. thought i mention that. wow this is long. thats all. if your refering to the commit description. when jolly jumpscares you. remains of a humans nerves come out of the mouth. thats all.

Reply:
The description is simply too long. I'm not sure it'll all display. Shorten it to something like  
> Starts, splits, resets. Set timer offset to -28.59. (by AITYuniverse and derric)

Reply:
updated